### DISCUSSION TITLE
Simple way to change the RGB values of the displayed GIF

### DISCUSSION THREAD

**Discussion by imWesentlichen** [bitbank2/AnimatedGIF#98] [General]
Hello, 

I want to display a GIF on a GC9A01 240x240px round Display and be able to change the RGB values of it on the left and the right side of the display independently. For example a dot that is green on the left side and red on the right side. 

Is there a easy way to do this? Can i maybe just display the one GIF as two GIFs (cut the one GIF to left and right side) at the same time and just change the color of each independently?

I tried to use the TFT_eSPI_memory example and got it to display my GIF but i cant figure out how to change the color of it.

Thank you for your work and that you provide this for free!

**Comment by bitbank2** [comment]
You can use the library as-is to do this. You can control the palette translation in the GIFDraw callback to behave any way you like.
